Mailnag/daemon/dbus.py

```diff
@@ -110,7 +110,7 @@ def _convert_mails(self, mails):
 			d['sender_addr'] = addr				# string (s)
 			d['account_name'] = m.account_name	# string (s)
 			d['id'] = m.id						# string (s)
-
+			d['strID'] = m.strID				# string (s)
```

Comment:
I think that was a mistake on my side, I think I tested that once to let the gnome-shell extension know about the ID (and open the specific mail directly when clicking on it) in the end it didn't really work. I guess it can be removed